libres/lib/job_queue/torque_driver.cpp

```diff
@@ -514,26 +531,49 @@ void *torque_driver_submit_job(void *__driver, const char *submit_cmd,
 static job_status_type
 torque_driver_get_qstat_status(torque_driver_type *driver,
                                const char *jobnr_char) {
-    char *tmp_file = (char *)util_alloc_tmp_file("/tmp", "enkf-qstat", true);
+    char *tmp_std_file =
+        (char *)util_alloc_tmp_file("/tmp", "ert-qstat-std", true);
+    char *tmp_err_file =
+        (char *)util_alloc_tmp_file("/tmp", "ert-qstat-err", true);
     job_status_type status = JOB_QUEUE_STATUS_FAILURE;
 
     {
         const char **argv = (const char **)util_calloc(1, sizeof *argv);
         argv[0] = jobnr_char;
 
-        util_spawn_blocking(driver->qstat_cmd, 1, (const char **)argv, tmp_file,
-                            NULL);
+        /* The qstat command might fail intermittently for acceptable reasons,
+           retry a couple of times with exponential sleep. ERT pings qstat
+           every second for every realization, thus the initial sleep time
+           is 2 seconds. */
+        int return_value = -1;
```

Comment:
I guess this `return_value` parameter can go directly into the while loop; ie. `int return_value = util_spawn_blocking(...)`

Reply:
It is needed to bootstrap the `while`-statement.